tests/utils/test_sampling_utils.py

```diff
@@ -28,3 +30,40 @@ def test_sample_points_on_plane() -> None:
     np.testing.assert_almost_equal(pts_residuals, np.zeros((num_points, 1)))
 
     assert pts.shape == (10, 3)
+
+
+def test_sample_random_directions() -> None:
```

Comment:
nit: could we add a brief 1-sentence docstring to document in prose what we are testing here?